src/main/java/com/amazon/opendistroforelasticsearch/security/compliance/ComplianceConfig.java

```diff
@@ -31,14 +31,7 @@
 package com.amazon.opendistroforelasticsearch.security.compliance;
 
 import java.nio.charset.StandardCharsets;
-import java.util.ArrayList;
```

Comment:
Please add `java.util.Collection` to the list of imports and remove `java.util.*`. Consider changing Java IDE preferences not to use `*` imports.